commit abc690952b94031ef044b1a0e63979c506632cf1
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Nov 27 16:33:31 2018 +0000

    ARM: dts: bcm283x: The lan7515 PHY node has moved
    
    The DT node describing the LAN7800s PHY has now moved inside an "mdio"
    node. Update the DT declarations accordingly.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
index 76039f81f17f..a925fa107a01 100644
--- a/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
+++ b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
@@ -1,4 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0
+#include <dt-bindings/net/microchip-lan78xx.h>
+
 / {
 	aliases {
 		ethernet0 = &ethernet;
@@ -21,13 +23,20 @@
 			ethernet: ethernet@1 {
 				compatible = "usb424,7800";
 				reg = <1>;
-				microchip,eee-enabled;
-				microchip,tx-lpi-timer = <600>; /* non-aggressive*/
-				/*
-				 * led0 = 1:link1000/activity
-				 * led1 = 6:link10/100/activity
-				 */
-				microchip,led-modes = <1 6>;
+
+				mdio {
+					#address-cells = <0x1>;
+					#size-cells = <0x0>;
+					eth_phy: ethernet-phy@1 {
+						reg = <1>;
+						microchip,eee-enabled;
+						microchip,tx-lpi-timer = <600>; /* non-aggressive*/
+						microchip,led-modes = <
+							LAN78XX_LINK_1000_ACTIVITY
+							LAN78XX_LINK_10_100_ACTIVITY
+						>;
+					};
+				};
 			};
 		};
 	};
@@ -36,9 +45,9 @@
 
 / {
 	__overrides__ {
-		eee = <&ethernet>,"microchip,eee-enabled?";
-		tx_lpi_timer = <&ethernet>,"microchip,tx-lpi-timer:0";
-		eth_led0 = <&ethernet>,"microchip,led-modes:0";
-		eth_led1 = <&ethernet>,"microchip,led-modes:4";
+		eee = <&eth_phy>,"microchip,eee-enabled?";
+		tx_lpi_timer = <&eth_phy>,"microchip,tx-lpi-timer:0";
+		eth_led0 = <&eth_phy>,"microchip,led-modes:0";
+		eth_led1 = <&eth_phy>,"microchip,led-modes:4";
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1362_abc690952b94031ef044b1a0e63979c506632cf1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1362_abc690952b94031ef044b1a0e63979c506632cf1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
